lesson06/assignment/html/grid.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="/css/style.css" />
+
+    <title>Assignment 06 - Bootstrap 4</title>
+</head>
+
+<body class="pl-5 pr-5">
+    <main class="main">
+        <!-- Nav Section -->
+        <nav class="navbar navbar-expand-sm bg-white navbar-light">
+            <a class="navbar-brand justify-content-start" href="/index.html">
+                <img src="/images/Your Home-1.png" alt="Company logo." width="120">
+            </a>
+            <ul class="navbar-nav ml-5">
+                <li class="pl-4 nav-item">
+                    <a class="nav-link" href="/index.html">Home</a>
+                </li>
+                <li class="pl-4 nav-item">
+                    <a class="nav-link" href="./about.html">About</a>
+                </li>
+                <li class="pl-4 nav-item">
+                    <a class="nav-link" href="./grid.html">Listings</a>
+                </li>
+                <li class="pl-4 nav-item">
+                    <a class="nav-link" href="./faq.html">FAQ</a>
+                </li>
+            </ul>
+        </nav>
+        <section class="mt-5">
+            <div class="container">
+                <div class="row">
+                    <!-- Listings Header -->
+                    <div class="col-md-12 bg-info p-2 mb-5">
+                        <p class="text-center text-white pt-2"><strong>HOT HOMES</strong></p>
```

Comment:
Changing pt-2 to pt-3 will vertically center the title in the blue box.  Another option would be to remove padding and use flexbox.